tensorboard/webapp/core/store/core_types.ts

```diff
@@ -35,3 +35,15 @@ export interface CoreState {
 export interface State {
   [CORE_FEATURE_KEY]?: CoreState;
 }
+
+export const initialState: CoreState = {
+  activePlugin: null,
+  plugins: {},
+  pluginsListLoaded: {
+    state: DataLoadState.NOT_LOADED,
+    lastLoadedTimeInMs: null,
+  },
+  reloadPeriodInMs: 30000,
+  reloadEnabled: false,
+  pageSize: 15,
```

Comment:
This is partly pre-existing, but I'd like to use this opportunity to comment. Compared to the other fields of this object, `pageSize` is especially unclear in meaning. I think it's better to add a comment to where the field is defined (L32 above) to describe what it is.